numpy-stubs/__init__.pyi

```diff
@@ -587,5 +590,33 @@ infty: float
 nan: float
 pi: float
 
+ALLOW_THREADS: int
+BUFSIZE: int
+CLIP: int
+ERR_CALL: int
+ERR_DEFAULT: int
+ERR_IGNORE: int
+ERR_LOG: int
+ERR_PRINT: int
+ERR_RAISE: int
+ERR_WARN: int
+FLOATING_POINT_SUPPORT: int
+FPE_DIVIDEBYZERO: int
+FPE_INVALID: int
+FPE_OVERFLOW: int
+FPE_UNDERFLOW: int
+MAXDIMS: int
+MAY_SHARE_BOUNDS: int
+MAY_SHARE_EXACT: int
+RAISE: int
+SHIFT_DIVIDEBYZERO: int
+SHIFT_INVALID: int
+SHIFT_OVERFLOW: int
+SHIFT_UNDERFLOW: int
+UFUNC_BUFSIZE_DEFAULT: int
+WRAP: int
+little_endian: int
+tracemalloc_domain: int
```

Comment:
This wasn't supposed to show up in the main namespace, release notes explicitly say `np.lib.tracemalloc_domain` - oh the namespace pollution :(